generic-styles/content-baked.less

```diff
@@ -636,9 +639,13 @@ figure {
     text-transform: uppercase;
   }
   p {
-    margin: 0 0 1rem 1.5rem;
+    margin: 1rem 0 1rem 1.5rem;
     color: #555;
   }
+
+  &[data-type="note"] {
```

Comment:
what is this and how is what its selecting different from the note styling above

Reply:
It's about Section Learning Objectives: https://openstax.org/books/physics/pages/15-1-the-electromagnetic-spectrum.
Wrapper `<div class="learning-objectives" data-type="note">:  `learning-objectives` are overriding the styling by `[data-type="note"]`, so it has `1.5rem` padding
Title: `[data-type="note"] > .os-title` overrides `.learning-objectives [data-type="title"] `, so it has the _new_ styling - additional padding. 

So I've added this selector to make normal learning objectives and noteish learning objectives look the same as before.